pom.xml

```diff
@@ -1110,7 +1110,7 @@ limitations under the License.]]></inlineHeader>
                 <requireMavenVersion>
                   <version>${mavenVersion}</version>
                 </requireMavenVersion>
-                <requireSameVersions />
+                <requireSameVersions></requireSameVersions>
               </rules>
```

Comment:
@vibe13 Seriously, what is going on here? I think it's a `sortpom:sort` bug, but why it is even running I can't figure out yet.